examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+            "[NeMo I 2020-05-11 02:34:57 callbacks:450] Evaluation time: 0.34247541427612305 seconds\n",
+            "[NeMo I 2020-05-11 02:35:02 callbacks:207] Finished epoch 96 in 0:00:08.679435\n",
+            "[NeMo I 2020-05-11 02:35:02 callbacks:199] Starting epoch 97\n",
+            "[NeMo I 2020-05-11 02:35:09 callbacks:224] Step: 2640\n",
+            "[NeMo I 2020-05-11 02:35:09 helpers:104] Loss: 0.0019539296627044678\n",
+            "[NeMo I 2020-05-11 02:35:09 helpers:110] training_batch_top@1:  100.0000\n",
+            "[NeMo I 2020-05-11 02:35:09 callbacks:239] Step time: 0.3111248016357422 seconds\n",
+            "[NeMo I 2020-05-11 02:35:10 callbacks:207] Finished epoch 97 in 0:00:08.322436\n",
+            "[NeMo I 2020-05-11 02:35:10 callbacks:199] Starting epoch 98\n",
+            "[NeMo I 2020-05-11 02:35:19 callbacks:207] Finished epoch 98 in 0:00:08.257718\n",
+            "[NeMo I 2020-05-11 02:35:19 callbacks:199] Starting epoch 99\n",
+            "[NeMo I 2020-05-11 02:35:21 callbacks:224] Step: 2680\n",
+            "[NeMo I 2020-05-11 02:35:21 helpers:104] Loss: 0.0006973743438720703\n",
+            "[NeMo I 2020-05-11 02:35:21 helpers:110] training_batch_top@1:  100.0000\n",
+            "[NeMo I 2020-05-11 02:35:21 callbacks:239] Step time: 0.24202775955200195 seconds\n",
+            "[NeMo I 2020-05-11 02:35:27 callbacks:207] Finished epoch 99 in 0:00:08.261154\n",
+            "[NeMo I 2020-05-11 02:35:27 callbacks:195] Done in 0:14:05.425577\n",
+            "[NeMo I 2020-05-11 02:35:27 callbacks:303] Saved checkpoint: ./myExps/checkpoints/quartznet3x1_an4/trainer-STEP-2700.pt\n",
+            "[NeMo I 2020-05-11 02:35:27 callbacks:468] Final Evaluation ..............................\n",
+            "[NeMo I 2020-05-11 02:35:27 helpers:273] ==========>>>>>>Evaluation Loss an4_dev: 1.313\n",
+            "[NeMo I 2020-05-11 02:35:27 helpers:275] ==========>>>>>>Evaluation Accuracy Top@1 an4_dev: 76.8421\n",
+            "[NeMo I 2020-05-11 02:35:27 callbacks:473] Evaluation time: 0.34503674507141113 seconds\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BB6s19pmxGfX",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now that we trained our embeddings, we shall extract these embeddings from using pretrained checkpoint present at `checkpoint_dir`. As we can see from the neural architecture, we extract the embeddings after `emb1` layer. \n",
+        "![Speaker Recognition Layers](./speaker_reco.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oSIDu6jkym66",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now use test manifest to get the embeddings. As we saw before let's create a new data\\_layer for test. And use previously instiated models and attach the DAGs"
```

Comment:
use the test
As we saw before let's -> As we saw before, let's
And use - > Use

Reply:
Done

Reply:
the 1st and 3rd changes were not done

Reply:
My bad, done now